.travis/test_runner.sh

```diff
@@ -10,13 +10,15 @@
 
 set -e
 
-TESTS="$1"
+TESTS="$@"
 COMMAND="coverage run manage.py test --noinput --failfast --traceback --verbosity=2"
 
 /moto-s3/env/bin/moto_server s3 &
 
 if [ -z ${COMMAND_OVERRIDE} ]; then
+    echo "Running tests: $TESTS"
     $COMMAND $TESTS
```

Comment:
fixed in https://github.com/dimagi/commcare-hq/pull/11568/files#diff-69316794bdf7ed7ed1be89f32cd69da0R19